hack/local-up-cluster.sh

```diff
@@ -473,20 +472,27 @@ function start_apiserver {
       priv_arg="--allow-privileged "
     fi
 
-    if [[ ${ADMISSION_CONTROL} == *"Initializers"* ]]; then
+    if [[ ${ENABLE_ADMISSION_PLUGINS} == *"Initializers"* ]]; then
         if [[ -n "${RUNTIME_CONFIG}" ]]; then
           RUNTIME_CONFIG+=","
         fi
         RUNTIME_CONFIG+="admissionregistration.k8s.io/v1alpha1"
     fi
 
-    if [[ ${ADMISSION_CONTROL} == *"PodPreset"* ]]; then
+    if [[ ${ENABLE_ADMISSION_PLUGINS} == *"PodPreset"* ]]; then
         if [[ -n "${RUNTIME_CONFIG}" ]]; then
             RUNTIME_CONFIG+=","
         fi
         RUNTIME_CONFIG+="settings.k8s.io/v1alpha1"
     fi
 
+    if [[ ${ENABLE_ADMISSION_PLUGINS} == *"EventRateLimit"* ]]; then
+        if [[ -n "${RUNTIME_CONFIG}" ]]; then
+            RUNTIME_CONFIG+=","
+        fi
+        RUNTIME_CONFIG+="eventratelimit.admission.k8s.io/v1alpha1"
```

Comment:
Actually, I am not a fan of this magic. This cluster should resemble real life and there activating the admission plugin is not enough either. I would rather remove the magical runtime configs in all other cases.

Reply:
make sense

Reply:
I think this should be kept as before. Because `EventRateLimit` is not enabled in local-up-cluster.sh, it is different from others.

Reply:
Revert it.